CMakeLists.txt

```diff
@@ -21,12 +21,20 @@ include(FetchContent)
 option(ASSERT_DECOMPOSE_BINARY_LOGICAL "Enables expression decomposition of && and || (this prevents short circuiting)" OFF)
 option(ASSERT_LOWERCASE "Enables assert alias for ASSERT" OFF)
 option(ASSERT_USE_MAGIC_ENUM "Use the MagicEnum library to print better diagnostics for enum classes" ON)
-option(ASSERT_USE_EXTERNAL_CPPTRACE "Controls whether cpptrace is gathered with FetchContent or fina_package" OFF)
+option(ASSERT_USE_EXTERNAL_MAGIC_ENUM "Controls whether magic_enum is gathered with FetchContent or find_package" OFF)
+option(ASSERT_USE_EXTERNAL_CPPTRACE "Controls whether cpptrace is gathered with FetchContent or find_package" OFF)
 option(ASSERT_STATIC "Specifies libassert as a static library" OFF)
 set(ASSERT_FAIL "" CACHE STRING "ASSERT_FAIL")
 
 option(ASSERT_BUILD_TESTS "Build tests" OFF)
 option(ASSERT_BUILD_BASIC_TEST "Build basic tests" OFF)
+
+set(ASSERT_MAGIC_ENUM_REPO "https://github.com/Neargye/magic_enum.git")
+set(ASSERT_MAGIC_ENUM_TAG "v0.9.4")
+
+set(ASSERT_CPPTRACE_REPO "https://github.com/jeremy-rifkin/cpptrace.git")
+set(ASSERT_CPPTRACE_TAG "v0.2.1")
```

Comment:
I don't love pulling these out into variables but does it serve a purpose for users of the library?

Reply:
Not yet but the best is to do :
```cmake
if(NOT DEFINED ASSERT_MAGIC_ENUM_REPO)
set(ASSERT_MAGIC_ENUM_REPO "https://github.com/Neargye/magic_enum.git")
endif()
```

like this user only need to do cmake -DASSERT_MAGIC_ENUM_REPO="****" to change it without modifying anything in your code. It s usefull to check a new version.